taipy/gui_core/_adapters.py

```diff
@@ -257,17 +257,17 @@ def _get_datanode_property(attr: str):
 
 
 class _GuiCoreScenarioProperties(_TaipyBase):
-    _SC_TYPES = {
-        "Config id": "string",
-        "Label": "string",
-        "Creation date": "date",
-        "Cycle label": "string",
-        "Cycle start": "date",
-        "Cycle end": "date",
-        "Primary": "boolean",
-        "Tags": "string",
+    _SC_TYPES: t.Dict[str, t.Dict[str, t.Union[str, bool]]] = {
```

Comment:
Now that I thing again about this, I think it makes far more sense to have a single dict:
```
_SC_PROPS: t.Dict[str, t.Dict[str, t.Union[str, bool]]] = {
    "Config id": {"attr": "config_id", "type": "string"},
     "Label": {"attr": "name", "type": "string"},
     "Creation date": {"attr": "creation_date", "type": "date",},
     ...
     "Cycle label": {"attr": "cycle.name", "type": "string""for_cycle": True},
     ...
     }
```

No need for _TYPES and _LABELS really, and less error prone.
Note that I renamed "cycle" to "for_cycle". Less ambiguous.